frontend2/src/components/CurrentUserProvider.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from "react";
+import { type UserPrivate } from "../utils/types";
+import { AuthStateEnum, type AuthState, CurrentUserContext } from "../contexts/CurrentUserContext";
+import { Api } from "../utils/api";
```

Comment:
After `https://github.com/battlecode/galaxy/pull/664` is merged, this will need to change to `import * as Api` :+1: